doc/radosgw/notifications.rst

```diff
@@ -84,6 +84,18 @@ which tells the client that it may retry later.
 .. tip:: To minimize the latency added by asynchronous notification, we 
    recommended placing the "log" pool on fast media.
 
+The following are the global configurations for asynchronous notifications:
```

Comment:
There's only one config value, so I think a list is overkill, unless there will be more in the future.  Suggest:

Persistent bucket notifications are managed by the following central configuration options: